.eslintrc.js

```diff
@@ -524,6 +524,16 @@ module.exports = {
       },
     },
 
+    /**
+     * object-prototype package specific rules
+     */
+    {
+      files: ['packages/object-prototype/**/*.js'],
+      rules: {
+        'mocha/handle-done-callback': 'off', // TODO: Find a way to disable all mocha rules
```

Comment:
Hooray for js config files?

```suggestion
        ...Object.fromEntries(Object.entries(require('eslint-plugin-mocha').rules).map(([k, v]) => [k, 'off']))
```

Reply:
You might need to update this to use `Object.keys().reduce()` since I'm pretty sure this config file will execute without the polyfills necessary for `Object.fromEntries()`

Reply:
If only we were using a newer version of Node.js that supported `Ojbect.fromEntires` 😉 

Joke aside, thanks - this is a great idea ❤️

Reply:
Once #59798 lands, I'll rebase this PR against `master` and use the newly created `allMochaRules` variable.

Reply:
Fixed